projects/notebooks/visualizations/rhods-notebooks-performance/models/lts.py

```diff
@@ -60,7 +61,7 @@ class Payload(matbench_models.ExclusiveModel):
     metadata: Metadata
     results: Results
     kpis: Optional[NotebookPerformanceKPIs]
-    regression_results: Optional[Any]
+    regression: Optional[List[matbench_models.RegressionResult]] = Field(default=None)
```

Comment:
can you call the field like the type? either `regression/Regression` or `regression_results/RegressionResult`, for consistency